examples/bq_file_load_benchmark/requirements.txt

```diff
@@ -0,0 +1,18 @@
+apache-beam==2.10.0
```

Comment:
can we unpin these dependencies? 
FYI when running `pip install -r requirements.txt` on a fresh python3 venv I get an error complaining about `apache-beam==2.10.0` not existing (But >=2.11 does). Not sure what's up with that.